test_image.sh

```diff
@@ -223,6 +225,39 @@ function test_docker_image_scripts_2 {
 	test_page "http://${CONTAINER_HOSTNAME}:${CONTAINER_HTTP_PORT}/test_docker_image_scripts_2.jsp" "TEST2"
 }
 
+function test_docker_image_with_playwright {
+	echo ""
+	echo "Waiting for application to be fully ready..."
+
+	while true
+	do
+		if docker logs "${CONTAINER_ID}" 2>&1 | grep -q "License registered for DXP Development" #name is not right at all
```

Comment:
@lucasmiranda0, can you please check if the portal is up and running just a few times to prevent the usage of `while true`?